.github/workflows/CompatHelper.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         run: which julia
         continue-on-error: true
       - name: Install Julia, but only if it is not already available in the PATH
-        uses: julia-actions/setup-julia@v1
+        uses: julia-actions/setup-julia@v2
         with:
           version: '1'
```

Comment:
Same here, maybe this needs to be `version: '2'`?

Reply:
That's the version of Julia that the action installs https://github.com/julia-actions/setup-julia